packages/web/app/views/programRegistry/ActivatePatientProgramRegistry.jsx

```diff
@@ -98,6 +98,7 @@ export const ActivatePatientProgramRegistry = ({ onClose, patientProgramRegistra
     );
 
     // Invalidate queries and close modal
+    queryClient.invalidateQueries(['PatientProgramRegistryConditions']);
```

Comment:
Actually an existing issue too where we don't reset the conditions on the activate form. The query keys are getting a refactor in the program registries epic but fixing here too.